### PR TITLE
LPS-134159 Provide an endpoint to retrieve the versions from a Process

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by naraandrade** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/ProcessVersionResourceImpl.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+			List::stream
+		).map(
+			SearchHit::getDocument
+		).map(
+			document -> document.getStrings("versions")
+		).flatMap(
+			List::stream
+		).map(
+			version -> {
+				ProcessVersion processVersion = new ProcessVersion();
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/ProcessVersionResourceImpl.java:88]: Hey @MarinhoFeliphe, please use a double brace initialization here. Thanks.

```java
new ProcessVersion() {
   {
        name = version;
   }
}
```

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/ProcessVersionResourceImpl.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+				processVersion.setName(version);
+
+				return processVersion;
+			}
+		).collect(
+			LinkedHashMap::new,
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/ProcessVersionResourceImpl.java:95]: Hey @MarinhoFeliphe , Why not use a `Collectors.toList()` instead?

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by naraandrade** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:forward

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:relevant

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**